synapse/config/tls.py

```diff
@@ -100,14 +143,12 @@ def is_disk_cert_valid(self, allow_self_signed=True):
             with open(self.tls_certificate_file, 'rb') as f:
                 cert_pem = f.read()
         except Exception:
-            logger.exception("Failed to read existing certificate off disk!")
-            raise
```

Comment:
removing the `raise` doesn't look right. (I'm also a bit unconvinced about changing from `exception` to `fatal`.)

Reply:
Have removed this code in favor of `self.read_file()`.